.env

```diff
@@ -0,0 +1,4 @@
+REACT_APP_API_ENDPOINT=https://api.codingcoach.io
+REACT_APP_AUTH_DOMAIN=codingcoach.eu.auth0.com
+REACT_APP_AUTH_CLIENT_ID=IxICRc28Q2waSi00bUBp294pAt0RmcRa
```

Comment:
Should this be checked in?

Reply:
If it uses in the client, it will be public anyway..

Reply:
Client Ids is fine, as @moshfeu pointed out, they will be available in the JS bundle anyways